cmd/controller/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
Please move to the build directory below.

Reply:
I removed the old dockerfiles and moved all related dockerfiles to `build/`.